src/lib/models/useFieldValidators.ts

```diff
@@ -26,7 +26,7 @@ export function useValidator({
         field: property,
         id: modelId,
     }) as Validator
-    if (propertyDetails.propertyType === SchemaFieldPropertyType.REFERENCE) {
+    if (propertyDetails.length) {
```

Comment:
I've no idea why this checked for `reference`?

Reply:
not sure eother, but io think it's my fault